portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase

```diff
@@ -1322,6 +1683,79 @@ definition {
 			fieldName = "Selección Única");
 	}
 
+	@description = "LPS-116866 - Verify the Submit Button Label can be translated to multiple languages"
+	@priority = "4"
+	test SubmitButtonCanBeTranslated {
```

Comment:
@leticiachiu, how about rename to `SubmitButtonLabelCanBeTranslated`?

I believe it'll a little bit more consistent with the existing UI.

![image](https://user-images.githubusercontent.com/6729287/127863470-8fed64f2-f73c-4fb2-bf50-307f7b1480dc.png)